.travis.yml

```diff
@@ -39,6 +39,7 @@ matrix:
 
 before_script:
   - rustup target add $ALT
+  - rustup component add clippy || echo "clippy not available"
```

Comment:
Should we do `echo "clippy not available"` here?

Reply:
I don't understand the question.  That's what it is doing.  Or maybe you meant to comment on the appveyor changes?

Reply:
I mean is it necessary to have `echo "clippy not available"`? Usually I only saw `rustup component add clippy` without the other part.

Reply:
If not, the command may fail because of missing clippy, and the build will stop.